bucket/coursier.json

```diff
@@ -7,8 +7,14 @@
     "hash": "4c554d97e7b17493735aed86db98753d253fd9ca095d9ff82a117153d61b78c3",
     "bin": "coursier.jar",
     "env_set": {
-        "COURSIER_HOME": "${env:LOCALAPPDATA}\\Coursier"
+        "COURSIER_CACHE": "$dir\\cache",
+        "COURSIER_BIN_DIR": "$dir\\bin"
     },
+    "env_add_path": "bin",
```

Comment:
`bin` field is already present, so is this required? We should keep only one of them.

Reply:
This is for installed app's bin, e.g., coursier install scala3 will put scala3.bat into bin folder.

Reply:
Understood